<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetStats</class>
 <widget class="QDialog" name="NetStats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <layout class="QFormLayout">
   <item row="0" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Sent:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="sentLabel">
     <property name="text">
      <string notr="true">0</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Received:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="recvLabel">
     <property name="text">
      <string notr="true">0</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Latency:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lagLabel">
     <property name="text">
      <string notr="true">0</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Tiles:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="tilesLabel">
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Tile Memory:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="tileMemoryLabel">
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Contexts:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Context Memory:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="contextMemoryLabel">
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="contextsLabel">
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NetStats</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NetStats</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
